HISTORY.md

```diff
@@ -2,6 +2,17 @@
 ## Unreleased
 ### Public API Change
 * `BackupableDBOptions::max_valid_backups_to_open == 0` now means no backups will be opened during BackupEngine initialization. Previously this condition disabled limiting backups opened.
+* `DBOptions::preserve_deletes == false` is a new option that allows one to specify that DB should not drop tombstones for regular deletes if they have sequence number larger than what was set by the new API call `DB::SetPreserveDeletesSequenceNumber(SequenceNumber seqnum)`.
```

Comment:
sorry to be pedantic. We usually don't break lines in this file since this text is copy/pasted into wiki/elsewhere, where each item needs to be one long line to show up as a single bullet point.

Also I think you're describing behavior for `preserve_deletes = true`. Or maybe you want to mention the default value is false? Might be least confusing to just say we're introducing `preserve_deletes`, without saying any value.

Reply:
No problem! Good pointed. Updated HISTORY.md file with that.